<?xml version="1.0" encoding="UTF-8"?><testrun name="MapBooksServiceFindTest" project="04_Testing_Nonfunctional" tests="4" started="4" failures="1" errors="0" ignored="0">
  <testsuite name="org.javacream.books.warehouse.business.MapBooksServiceFindTest" time="0.0">
    <testcase name="testFindBookByIsbn" classname="org.javacream.books.warehouse.business.MapBooksServiceFindTest" time="0.0">
      <failure>java.lang.AssertionError: Book must be available&#13;
	at org.junit.Assert.fail(Assert.java:93)&#13;
	at org.junit.Assert.assertTrue(Assert.java:43)&#13;
	at org.javacream.books.warehouse.business.MapBooksServiceFindTest.testFindBookByIsbn(MapBooksServiceFindTest.java:39)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNullIsbn" classname="org.javacream.books.warehouse.business.MapBooksServiceFindTest" time="0.0"/>
    <testcase name="testNullTitle" classname="org.javacream.books.warehouse.business.MapBooksServiceFindTest" time="0.0"/>
    <testcase name="testUnknownIsbn" classname="org.javacream.books.warehouse.business.MapBooksServiceFindTest" time="0.0"/>
  </testsuite>
</testrun>
